docs/index.asciidoc

```diff
@@ -141,7 +142,15 @@ E.g: `/%{myfield}/`, `/test-%{myfield}/` are not valid paths
 If `append`, the file will be opened for appending and each new event will be written at the end of the file.
 If `overwrite`, the file will be truncated before writing and only the most recent event will appear in the file.
 
-[id="plugins-{type}s-{plugin}-common-options"]
+[id="plugins-{type}s-{plugin}-stale_cleanup_interval"]
+===== `stale_cleanup_interval`
+
+  * Value type is <<number,number>>
+  * Default value is `10`
+
+Defines the interval, in seconds, between the stale files cleanup runs (in seconds).
+The stale files cleanup cycle closes inactive files (i.e files not written to since the last cycle).
+
```

Comment:
Nice job describing the new option and its use! One request: Please move the new option to alphabetical order rather than operational order. This standard makes it easier for users to scan and find options.